FileDelete/FileDelete/FileDelete.inf

```diff
@@ -4,29 +4,84 @@
 
```

Comment:
Please remove this file changes from the PR :)

Reply:
That's OK. But I wonder whether the following issues would appear or not when using your original .inf file. 

![image](https://user-images.githubusercontent.com/57588843/181792679-98970a4b-b6f5-401c-9027-e0f09b424bd3.png)

I am attempting to remove this change. Sorry that I am not familiar with the operation. Please wait for a while.

Reply:
Everything is ok :)
No pressure, take your time.

Maybe the inf should be deleted because it isn't relevant here. What do you think?

Reply:
Yeah, I agree.